apps/qubit/modules/informationobject/templates/uploadFindingAidSuccess.php

```diff
@@ -0,0 +1,44 @@
+<?php decorate_with('layout_2col.php') ?>
+
+<?php slot('sidebar') ?>
+
+  <?php include_component('repository', 'contextMenu') ?>
+
+<?php end_slot() ?>
+
+<?php slot('title') ?>
+
+  <h1><?php echo $resource->title ?></h1>
+
+<?php end_slot() ?>
+
+<?php slot('content') ?>
+
+  <?php echo $form->renderGlobalErrors() ?>
+
+  <?php echo $form->renderFormTag(url_for(array($resource, 'module' => 'informationobject', 'action' => 'uploadFindingAid'))) ?>
+
+    <?php echo $form->renderHiddenFields() ?>
+
+    <div id="content">
+
+      <fieldset class="collapsible">
+
+        <legend><?php echo __('Upload finding aid') ?></legend>
+
+        <?php echo $form->file->label(__('PDF file'))->renderRow() ?>
```

Comment:
Not sure if you can pass HTML attributes to `renderRow()` but it'd be nice to use `accept` as mentioned earlier.